config/areas/lab/dialogs.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use Kirby\Panel\Lab\Responses;
+
+return [
+	'lab.errors' => [
+		'pattern' => 'lab/errors/(:any?)',
+		'load'    => fn (string|null $type = null) => Responses::errorResponseByType($type)
```

Comment:
`Nitpick:`

Only if you like that style too

```suggestion
		'load'    => Responses::errorResponseByType(...)
```

Reply:
I tried that but it doesn't work. It seems to be trying to bind $this for some which leads to an error.

Reply:
@bastianallgeier Indeed getting the same error. I have trouble debugging it as our Panel does swallow the PHP details/stack trace of the exception/error completely. This might not be caused by this PR, but since this PR kind of addresses these problems around errors in the Panel, could we improve this as well? Those PHP error messages are often worthless without more details.

Reply:
As a next step, I really want to build a better error dialog with the option to debug errors with more details. But I think we should get this done first and move on from here.

Reply:
Ok